libs/domains/services/feature/src/lib/hooks/use-count-services/use-count-services.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery } from '@tanstack/react-query'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseCountServicesProps {
+  environmentId?: string
+}
+
+export function useCountServices({ environmentId }: UseCountServicesProps) {
```

Comment:
Perhaps it would be better to have it in the Environments domain, but I'm not sure

Reply:
I think it's fine